lib/elixir/lib/map.ex

```diff
@@ -604,7 +604,7 @@ defmodule Map do
 
   ## Examples
 
-      iex> Map.update(%{a: 1}, :a, 13, &(&1 * 2))
+      iex> Map.update(%{a: 1}, :a, 13, fn existing -> existing * 2 end)
```

Comment:
```suggestion
      iex> Map.update(%{a: 1}, :a, 13, fn value -> value * 2 end)
```

Reply:
`value` reads better IMO

Reply:
How about `existing_value`? There is nothing wrong with being extra verbose imo.

Reply:
Yes. But use `current_value`. That's what's being used in the example in `Map.get_and_update/3`

Reply:
and I would replace the captures in the second example as well.